### PR TITLE
Add SAP SIDs to general information page

### PR DESCRIPTION
**Pull request by rvsia**
https://issues.redhat.com/browse/RHCLOUD-9650

**A host with SAP**

sids

**A host without SAP**

image

cc @iphands

### CODE REVIEW

File: packages/inventory-general-info/src/SystemCard.js [hunk 1]
```diff
@@ -101,6 +102,17 @@ class SystemCard extends Component {
                                 </Fragment>
                             ), size: 'md'
                         },
+                        {
+                            title: 'SAP',
+                            value: properties.sap_sids ? `${properties.sap_sids.length} identifiers` : 0,
+                            target: 'sap_sids',
+                            onClick: () => {
+                                handleClick(
+                                    'SAP SIDs',
```
Comment (karelhala) [inline packages/inventory-general-info/src/SystemCard.js:111]: Can you change this to `SAP IDs (SID)`  so we match the rest of platform?

File: packages/inventory-general-info/src/selectors.js [hunk 2]
```diff
@@ -27,8 +28,9 @@ export const propertiesSelector = ({
         ...mount_point && safeParser(mount_point, 'mountpoint'),
         ...options && safeParser(options, 'options'),
         ...type && safeParser(type, 'mounttype')
-    })
-    )
+    }),
+    ),
+    sap_sids
```
Comment (karelhala) [inline packages/inventory-general-info/src/selectors.js:33]: I'd rename this to `sapIds: sap_sids` so we don't have to disable cammel case all the time.

**Review by karelhala — COMMENTED** [review]
Looking good! Just a few minor improvements and we are golden.

**Comment by rvsia** [comment]
@karelhala feedback addressed! :+1:

**Review by karelhala — APPROVED** [review]

**Pull request merged**